meta-psa/recipes-security/psa-trusted-storage-linux/psa-trusted-storage-linux_1.0.0.bb

```diff
@@ -32,3 +42,14 @@ RDEPENDS_${PN} = "\
     ecryptfs-utils \
     keyutils \
 "
+
+SYSTEMD_PACKAGES = "${PN}"
+SYSTEMD_SERVICE_${PN} = "psa-ecryptfs.service"
+
+do_install_append() {
+ 
+    if ${@bb.utils.contains('DISTRO_FEATURES','systemd','true','false',d)}; then
```

Comment:
Why are putting the service and script files installation in a separate `do_install` function?
These lines should go to the `do_install` function above.